tools/issue-labeler/src/IssueLabelerService/TraigeRag.cs

```diff
@@ -205,6 +213,19 @@ public double GetHighestScore(IEnumerable<Issue> issues, IEnumerable<Document> d
 
             return highestScore;
         }
+
+        public string LabelsFilter(string[] labels)
+        {
+            if (labels != null && labels.Length >= 2)
```

Comment:
```suggestion
            if (labels?.Take(2).Count() == 2)
```

 We will need to be able to handle other schemes from non-Azure SDK repositories.  We should move `2` into some kind of `minimum label predictions for answer` config eventually.